nautobot/ui/src/components/Layout.js

```diff
@@ -8,40 +9,20 @@ import {
     Button,
 } from "@nautobot/nautobot-ui";
 
-// import { useEffect } from "react";
-
-// import LoadingWidget from "@components/LoadingWidget";
 import RouterLink from "@components/RouterLink";
 import SidebarNav from "@components/SidebarNav";
-// import { useGetSessionQuery, useGetUIMenuQuery } from "@utils/api";
-import { useSelector } from "react-redux";
 import { isLoggedInSelector } from "@utils/store";
 
 export default function Layout({ children }) {
     const isLoggedIn = useSelector(isLoggedInSelector);
 
-    // const { isSuccess: menuLoaded, refetch: refetchMenu } = useGetUIMenuQuery();
-
-    let toRender = children;
-
-    // TODO: showing the loading widget while the menu is loading is breaking the login route if not logged into the backend server
-    // if (!sessionLoaded || !menuLoaded || sessionInfo === undefined)
-    //     toRender = <LoadingWidget name="application" />;
-
+    /** Invalidate the newui cookie and reload the page in order to return to the legacy UI. */
```

Comment:
Is this js doc syntax or should it be `//`?

Reply:
https://jsdoc.app/about-getting-started.html